Gemfile

```diff
@@ -8,6 +8,7 @@ source 'http://rubygems.org'
 gemspec
 
 group :development do
+  gem 'concurrent-ruby', '= 1.3.4'
```

Comment:
Was this needed for something?

Reply:
yes. `concurrent-ruby` has changed its dependencies since `1.3.5` which raises the following error when launching the specs
```
uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger
```
The solution is either to upgrade rails dummy to 7.1 or to block concurrent-ruby version to 1.3.4. I chose the simplest one.

https://stackoverflow.com/questions/79360526/uninitialized-constant-activesupportloggerthreadsafelevellogger-nameerror

Reply:
Otherwise, if you don't like this workaround, I can remove this line from the `Gemfile` and add this to `spec/dummy/config/boot.rb`
```ruby
require "logger"
```
Would it be better?